src/api/spec/features/beta/webui/requests_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'browser_helper'
+
+RSpec.describe 'Requests', :beta, :js, :vcr do
```

Comment:
```suggestion
RSpec.describe 'Requests', :vcr do
```
`:beta` and `:js` are implicit from the spec path. They can be removed.

Also, if `:vcr` is needed, we are missing the cassettes.

Reply:
True

Reply:
`:vcr` is needed and you need the cassettes. The specs error out because there is no backend

Reply:
Some other cassettes recorded for other tests were used to run this new spec. That's why before, these tests passed adding only the `:vcr` flag. I modified the contexts and created new cassettes.

Additionally, I changed `click_on` to `click_button`.

Now tests should pass. Please take a look.